ci/taos/webapp/pr-handler-proceed.php

```diff
@@ -101,11 +101,11 @@ function goBack() {
 }
 
 if ( $id == $ba_id && $password == $ba_password) {
-    echo ("ID and Pasword is correct.<br>");
+    echo ("Oooops. ID and Pasword is correct.<br>");
```

Comment:
It's just simple question.. Why are you shouting oops here?
Anyway LGTM.

Reply:
It will be fixed with next PR. :)